bioblend/galaxyclient.py

```diff
@@ -7,16 +7,32 @@
 """
 import base64
 import json
+import os
 
-import poster
 import requests
-from six.moves.urllib.request import Request, urlopen
+from requests_toolbelt import MultipartEncoder
 import six
 
 from .galaxy.client import ConnectionError
 
 
 class GalaxyClient(object):
+
+    def attach_file(self, path, name=None):
```

Comment:
Not sure this function belongs to this class (does not use self), it may probably be moved to `bioblend/__init__.py`.